.github/workflows/playwright_comment.yml

```diff
@@ -0,0 +1,72 @@
+name: Comment on the PR with Playwright results
+
+on:
+  workflow_run:
+    workflows: ["Playwright Tests"]
+    types:
+      - completed
+permissions: {}
```

Comment:
Is this line even needed?

Reply:
It reduces the permissions. They are read-only across everything by default. Slightly more secure in terms of avoiding data exfiltration of some sort by one of the called actions.